doc/user/content/transform-data/patterns/temporal-filters.md

```diff
@@ -9,25 +9,48 @@ menu:
     parent: 'sql-patterns'
 ---
 
-A _temporal filter_ is a `WHERE` or `HAVING` clause which uses the [`mz_now()`](/sql/functions/now_and_mz_now) function.
-This function returns Materialize's current virtual timestamp, which works to keep up with real time as data is processed.
-Applying a temporal filter reduces the working dataset, saving memory resources and focusing results on the recent past.
-
-Here is a typical temporal filter that considers records whose timestamps are within the last 5 minutes.
+A **temporal filter** is a query condition/predicate that filters data based on
+a time-related (i.e., temporal) condition. Using a temporal filter reduces the
+working dataset, saving memory resources and focusing on results that are within
+the timeframe specified by the filter.
+
+In Materialize, you implement temporal filters using the
+[`mz_now()`](/sql/functions/now_and_mz_now) function (which returns
+Materialize's current virtual timestamp and keeps up with real time) in a
+`WHERE` or `HAVING` clause; specifically, you compare
+[`mz_now()`](/sql/functions/now_and_mz_now) to a numeric or timestamp column
+expression. As [`mz_now()`](/sql/functions/now_and_mz_now) progresses (every
+millisecond), records that fall outside the timeframe are retracted from the
+working dataset. That is, when using temporal filters, Materialize maintains (in
+memory) all future retractions of the data that is currently within the
+timeframe and applies the retraction when records fall out of the timeframe.
+
+For example, the following temporal filter reduces the working dataset to those
+records whose event timestamp column (`event_ts`) is within the last 5 minutes.
```

Comment:
I think .. technically rather than "within the last 5 minutes" the correct thing to say is "is no more than 5 minutes ago". If the timestamp is in the future, we'll include it also. It's not uncommon to do a filter like this but instead of `event_ts` it is `remove_ts`, with a future time for removal.

Reply:
Will update!  Thanks ever so much!